### PR TITLE
type alias change

### PR DESCRIPTION
**Pull request by angelwen08**

### CODE REVIEW

File: src/jsonTraverse.ts [hunk 1]
```diff
@@ -91,7 +91,12 @@ export function traverse(node: Node, parentUid: string, parentContainer: YamlMod
         }
 
         if (myself.type === 'type alias') {
-            myself.syntax = { content: 'type ' + myself.name + ' = ' + parseTypeDeclarationForTypeAlias(node.type) };
+            let typeArgumentsContent = parseTypeArgumentsForTypeAlias(node);
+            if (typeArgumentsContent) {
+                myself.syntax = { content: 'type ' + myself.name + typeArgumentsContent + ' = ' + parseTypeDeclarationForTypeAlias(node.type) };
```
Comment (wuhanyumsft) [inline src/jsonTraverse.ts]: use ` for better formating

File: src/jsonTraverse.ts [hunk 2]
```diff
@@ -357,7 +383,7 @@ function parseCommonTypeInfo(typeInfo: ParameterType, type: string, seperator: s
         } else if (item.value) {
             return `"${item.value}"`;
         } else {
-            return 'Object';
+            return parseUserDefinedType(item);
         }
```
Comment (wuhanyumsft) [inline src/jsonTraverse.ts]: add a space after character ':' in parseUserDefinedType method.

File: src/jsonTraverse.ts [hunk 3]
```diff
@@ -357,7 +383,7 @@ function parseCommonTypeInfo(typeInfo: ParameterType, type: string, seperator: s
...
     }).join(seperator);
```
Comment (wuhanyumsft) [inline src/jsonTraverse.ts]: type = child.name + ':' + `"${child.type.value}"`;
->

type = `${child.name}: ${child.type.value}`;

Comment (wuhanyumsft) [reply src/jsonTraverse.ts]: try to use `` as much as possible + string concat is very hard to read.

File: src/jsonTraverse.ts (hunk 1 above)
Comment (angelwen08) [reply src/jsonTraverse.ts]: For here, I prefer to keep current format for better understanding.

**Pull request merged**